src/shared/ml-thirdparty-client/hapi-plugin.ts

```diff
@@ -28,6 +28,11 @@
 import Client from '~/shared/ml-thirdparty-client'
 import { Plugin, Server } from '@hapi/hapi'
 import { Options } from './options'
+import config from '~/lib/config'
+// import { logger } from '~/shared/logger'
+import { Simulator } from '../ml-thirdparty-simulator'
```

Comment:
having Simulator branch code in deployment isn't a good idea, we should get rich of it in the future....